src/SME.SGP.Aplicacao/Commands/PendenciaAusenciaFechamento/ExecutarVerificacaoPendenciaAusenciaFechamento/ExecutarVerificacaoPendenciaAusenciaFechamentoCommandHandler.cs

```diff
@@ -0,0 +1,157 @@
+                    foreach (var professorTurma in professoresTurma)
+                    {
+                        var obterComponenteCurricular = componentes.FirstOrDefault(c => long.Parse(c.Codigo) == professorTurma.DisciplinaId);
+                        if (obterComponenteCurricular != null)
+                        {
+                            var verificaFechamento = await mediator.Send(new VerificaFechamentoTurmaDisciplinaPorIdCCEPeriodoEscolarQuery(turma.Id, long.Parse(obterComponenteCurricular.Codigo), periodoEscolar.Id));
+                            if (!verificaFechamento)
+                            {
+                                if(professorTurma.ProfessorRf != "")
+                                {
+                                    if (!await ExistePendenciaProfessor(turma.Id, long.Parse(obterComponenteCurricular.Codigo), professorTurma.ProfessorRf))
+                                        await IncluirPendenciaAusenciaFechamento(turma.Id, long.Parse(obterComponenteCurricular.Codigo), professorTurma.ProfessorRf, periodoEscolar.Bimestre, obterComponenteCurricular.Descricao, obterComponenteCurricular.LancaNota);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            return true;
+        }
+
+        private async Task<bool> ExistePendenciaProfessor(long turmaId, long componenteCurricularId, string professorRf)
+           => await mediator.Send(new ExistePendenciaProfessorPorTurmaEComponenteQuery(turmaId,
+                                                                                       componenteCurricularId,
+                                                                                       professorRf,
+                                                                                       TipoPendencia.AusenciaFechamento));
+
+        private async Task IncluirPendenciaAusenciaFechamento(long turmaId, long componenteCurricularId, string professorRf, int bimestre, string componenteCurricularNome, bool lancaNota)
+        {
+            var turma = await mediator.Send(new ObterTurmaComUeEDrePorIdQuery(turmaId));
+
+            var escolaUe = $"{turma.Ue.TipoEscola.ShortName()} {turma.Ue.Nome} ({turma.Ue.Dre.Abreviacao}) - Turma {turma.Nome}";
+            var titulo = $"Processo de fechamento não iniciado de {componenteCurricularNome} no {bimestre}º bimestre - <i>{escolaUe}</i>";
+
+            var descricao = $"<i>O componente curricular <b>{componenteCurricularNome}</b> ainda não teve o processo de fechamento do <b>{bimestre}º bimestre</b> iniciado na <b>{escolaUe}</b></i>";
+
+            var instrucao = "";
+            if (lancaNota)
+            {
+                instrucao = "<i style='color:#FF0000';>Você precisa acessar a tela de Notas e Conceitos e registrar a avaliação final dos estudantes para que o processo seja iniciado.</i>";
```

Comment:
essa cor vermelha no texto tinhamos combinado de não usar. Como padrão usamos o bold na instrução pra ter um destaque, isso já é aplicado no obter do dashboard, não precisa aplicar o estilo aqui
```suggestion
                instrucao = "Você precisa acessar a tela de Notas e Conceitos e registrar a avaliação final dos estudantes para que o processo seja iniciado";
```